content/javascript/concepts/operators/operators.md

```diff
@@ -68,12 +81,95 @@ console.log(number);
 
 ## Comparison Operators
 
-- `==` equal to
-- `===` equal value and equal type
-- `!=` not equal
-- `!==` not equal value or not equal type
-- `>` greater than
-- `<` less than
-- `>=` greater than or equal to
-- `<=` less than or equal to
-- `?` ternary operator
+These operators compare values and return a boolean value of `true` or `false`.
+
+- `==`: Returns `true` or `false` based on whether the value of two operands are equal.
```

Comment:
Do we want to put a warning that there is a difference between `==` and `=`? I feel like some people might confuse the two together.

Reply:
@HishamT Excellent suggestion, I'll add a note! 😄